.github/workflows/test.yaml

```diff
@@ -3,22 +3,6 @@ name: Tests
 # Execute workflow for each PR and with each merge to the trunk
 on:
   workflow_dispatch:
-    inputs:
-      zkvyper-version:
-        type: string
-        description: "Prebuilt zkvyper version, (repo: https://github.com/matter-labs/era-compiler-vyper/releases)"
-        required: false
-        default: "1.5.0"
-      vyper-version:
-        type: string
-        description: "vyper version, (repo: https://github.com/vyperlang/vyper/releases)"
-        required: true
-        default: "0.4.1"

```

Comment:
Oh, we don't support that one yet.

Reply:
Yeah, I don't know why it was here for the manual trigger, I removed that anyway. Ready to be merged now.